snuba/migrations/groups.py

```diff
@@ -169,6 +172,11 @@ def __init__(
         storage_sets_keys={StorageSetKey.METRICS_SUMMARIES},
         readiness_state=ReadinessState.PARTIAL,
     ),
+    MigrationGroup.PROFILE_CHUNKS: _MigrationGroup(
+        loader=ProfileChunksLoader(),
+        storage_sets_keys={StorageSetKey.PROFILE_CHUNKS},
+        readiness_state=ReadinessState.PARTIAL,
```

Comment:
Does the ClickHouse cluster already exist in SaaS (US + DE) and S4S? If the table is suppose to live the profiles cluster, has the `storage_set` been added there? If this isn't done yet, we should set the readiness state to `ReadinessState.LIMITED` for now so that GoCD doesn't try to run these migrations on a cluster that doesn't exist.

Reply:
The ClickHouse cluster we'd use exists in all environments (`us`, `de`, `s4s`, all STs).

I also have https://github.com/getsentry/ops/pull/10526 to configure it for `us` and `s4s`.

Reply:
+1

Reply:
which cluster are you using @phacops ?

Reply:
The cluster dedicated to profiling.